### ISSUE TITLE
Question: Increase the animation duration to open the bottom sheet

### ISSUE THREAD

**Issue by thomasdittmar** [osdnk/react-native-reanimated-bottom-sheet#348]
Love the lib.

This is more a question.

How can i increase the open animation duration?

**Comment by rishabh876** [comment]
I am also looking for this. Bottom sheet open animation is too fast for my taste.

**Comment by ManuViola77** [comment]
Yes! I am also trying to slow down the animation without success for now. I was told to use the `springConfig` parameter but I'm not noticing any change, any ideas?
